tests/v2/test_waku_store.nim

```diff
@@ -36,12 +36,12 @@ procSuite "Waku Store":
 
     proto.setPeer(listenSwitch.peerInfo)
 
-    var subscriptions = newTable[string, MessageNotificationSubscription]()
-    subscriptions["test"] = subscription
+    var subscriptions = newTable[string, MessageNotificationSubscription]() # listener
```

Comment:
All the changes in this file seems unnecessary

Reply:
Sure, addressed